deploy/template.yaml

```diff
@@ -3003,6 +3101,509 @@ Resources:
       ComparisonOperator: GreaterThanThreshold
       TreatMissingData: notBreaching
 
+  ####################################################################
+  #                                                                  #
+  # Canary Alarms                                                    #
+  #                                                                  #
+  ####################################################################
+
+  JourneyEngineStepFunctionExecutionFailedCanaryAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
```

Comment:
start with false so they don't fire to slack for production where it is set to not all at once?